public/components/security/roles-mapping/roles-mapping.tsx

```diff
@@ -49,15 +50,15 @@ export const RolesMapping = () => {
     if (!rolesLoading && (roles || [])) {
```

Comment:
The null coalescing operator  (`|| []`) can be avoided as follows:
```
const rolesArray = roles || [];
    if (!rolesLoading && rolesArray.length) {
      const _rolesObject = rolesArray.reduce(
        (rolesObj, role) => ({ ...rolesObj, [role.id]: role.name }),
        {}
        {},
      );
      setRolesEquivalences(_rolesObject);
    }
```

Reply:
I did not touch this code.

I am not sure if your proposal could cause something doesn't work as expected.

The condition:
```js
&& rolesArray.length
```
causes the if block to be accessed when there are more than 0 elements. I don't know if the `setRolesEquivalences` could be set with an empty array. We should analyze it.